container/rules/docker_toolchains.bzl

```diff
@@ -225,9 +190,9 @@ language_tool_layer_ = rule(
 )
 
 def language_tool_layer(**kwargs):
-    """A wrapper around attrs in container_image, download_pkgs and install_pkgs rules.
+    """A wrapper around attrs in container_image and install_pkgs rules.
 
-    Downloads and installs debian packages using
+    Installs debian packages using
```

Comment:
Why did you change the docs? Does this rule no longer do the downloading of packages?

Reply:
I will extract the downloading of packages from ```language_tool_layer``` into a new rule, which will be used by ```toolchain_container``` rule.

Reply:
I dont think its right to extract the downloading bit from this rule but leave the installing bit.